hollow/src/main/java/com/netflix/hollow/api/codegen/HollowCodeGenerationUtils.java

```diff
@@ -293,4 +297,49 @@ public static String getJavaScalarType(FieldType fieldType) {
         }
         throw new IllegalArgumentException("Java scalar type is not known for FieldType." + fieldType.toString());
     }
-}
+
+    private static final Set<String> booleanMethodPrefixes = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
+            "is", "has", "do", "should", "was", "contains", "enable", "disable", "get")));
+    public static Set<String> getBooleanMethodPrefixes() { return Collections.unmodifiableSet(booleanMethodPrefixes); }
+
+    /**
+     * Rules: prepend "get" / "is" + upper case first char of field name
+     *
+     * boolean/Boolean field:
+     *    - has a boolean prefix (@see {@link #booleanMethodPrefixes}), just return it; otherwise, prepend "get" + upper case first char
+     *
+     *      boolean isPrimary - isPrimary()
+     *      boolean hasStreams - hasStreams()
+     *      boolean playable - getPlayable()
```

Comment:
I was going with that as well originally but "value"  field became isValue() vs getValue().